src/librustc_yk_sections/Cargo.toml

```diff
@@ -13,6 +13,8 @@ test = false
 [dependencies]
 rustc = { path = "../librustc" }
 rustc_yk_link = { path = "../librustc_yk_link" }
-ykpack = { git = "https://github.com/softdevteam/yk" }
+#ykpack = { git = "https://github.com/softdevteam/yk" }
+ykpack = { path = "/home/vext01/research/yk/ykpack" }
```

Comment:
I know why this is here, but we need to work out something better than leaking private pathnames out each time. Unfortunately I'm not sure what that better thing might be. We could point `ykpack` at a `dev` branch or something?

Reply:
It doesn't bother me too much. There's nothing secret in the path name. I'd rather not complicate the git situation more than necessary.